package.json

```diff
@@ -41,11 +41,12 @@
   "peerDependencies": {
     "vue": "^2.5.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "axios": "^0.18.0"
+  },
```

Comment:
why `axios` changed to dependencies?

Reply:
Could potentially just use native fetch, tho fetch isn't supported in all browsers. Any other preferences

Reply:
I mean all modules imported are in `devDependencies ` for now.

Reply:
Hmm. I think if there is going to be to support a http client natively to this project, then the client used internally needs to be either a dependency or a peer dependency. Adding as dev dependency will eventually result in it not being installed by `npm install vue-material`, which may cause confusion for users.

Also open to replacing this with `XMLHttpRequest` (preferrably not, since this has really fallen out) or `fetch` (with a polyfill installed as dependency), depending on your team's preference for direction

Reply:
You don't need to install the modules which vue-material used because the `vue-material` module you importing is production that already compiled with all modules imported to it.

Reply:
For example, you can import `vue-material` without problem if you don't install `date-fns` and `popper.js`.